contracts/mocks/DCAHub/DCAHubPositionHandler.sol

```diff
@@ -12,6 +12,14 @@ contract DCAHubPositionHandlerMock is DCAHubPositionHandler, DCAHubConfigHandler
     IDCAPermissionManager _permissionManager
   ) DCAHubConfigHandlerMock(_immediateGovernor, address(1), _oracle) DCAHubPositionHandler(_permissionManager) {}
 
+  function setActiveSwapIntervals(
```

Comment:
I think it would make sense to add this to the ParametersMock, so that we can use it from other places also